src/utils/utils.c

```diff
@@ -160,11 +159,9 @@ bool cmp_prefix (const char* __restrict__ prefix,
     {
         const char* const u_len_end = ((const char*)u_len) + sizeof (size_t);
         bool no_overlap =
-            ((prefix + strlen (prefix) <= (char*)u_len)
-             || (u_len_end <= prefix))
+            ((prefix + strlen (prefix) <= (char*)u_len) || (u_len_end <= prefix))
             && ((full + strlen (full) <= (char*)u_len) || (u_len_end <= full))
-            && ((delim + strlen (delim) <= (char*)u_len)
-                || (u_len_end <= delim));
+            && ((delim + strlen (delim) <= (char*)u_len) || (u_len_end <= delim));
 
         if (!no_overlap) {
             DPRINTF ("DYAD UTIL: buffers overlap.\n");
```

Comment:
Why are these logging different than the login we do in DYAD like DYAD_LOG_INFO

Reply:
Not sure. That's been there since I started on DYAD.